examples/web/RunAnywhereAI/src/styles/components.css

```
@@ -1665,6 +1684,129 @@
+  text-align: left;
+  transition: background var(--duration-fast), border-color var(--duration-fast);
+}
+.hf-repo-row:hover {
+  background: var(--bg-tertiary);
+  border-color: var(--border-strong);
+}
+.hf-repo-row__info { flex: 1; min-width: 0; }
+.hf-repo-row__name {
+  font-size: var(--font-size-sm);
+  font-weight: var(--font-weight-medium);
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)